client/src/components/NavDots.jsx

```diff
@@ -8,32 +8,44 @@ class NavDots extends React.Component {
       currentPhoto: this.props.currentPhoto,
       active: [0, 1, 2, 3, 4]
     }
+    this.setClasses = this.setClasses.bind(this);
 /*     this.animate = this.animate.bind(this); */
   }
 
+  setClasses() {
+    let goodRange = [];
+    let index = this.state.currentPhoto
+    if (this.state.length >= 5) {
+      if (index === 0) {
+        // finish implementation later
```

Comment:
I think it's a good practice to append these types of comments with:

//TODO: do this.. later

Then also follow it up by creating an "issue" in your github showing that this is on the actual to do list.
We can also maintain these types of comments easier in a development branch, and not merge these types of things in to the main branch.